tools/utility.hpp

```diff
@@ -185,4 +185,47 @@ bool isChassisOn()
     // In any error condition assuming chassis is off.
     return false;
 }
+
+using ObjectMap =
+    std::map<sdbusplus::message::object_path,
+             std::map<std::string, std::map<std::string, std::variant<bool>>>>;
+
+/**
+ * @brief An API to call PIM
+ *
+ * This API calls notify method of PIM to update the property value.
+ *
+ * @param[in] objectMap - Map of object path, interface, property and its value.
+ */
+void callPIM(ObjectMap objectMap)
+{
+    try
+    {
+        for (const auto& objectKeyValue : objectMap)
```

Comment:
The key value pair can be extracted here itself, do we need another variable and the extract call?

Reply:
Got it.